### PR TITLE
Updating HDInsight Apps templates to use the V2 version

### PR DESCRIPTION
**Pull request by bharathsreenivas**

### CODE REVIEW

**Comment by azurecla** [comment]
Hi __@bharathsreenivas__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <span>
        In order for us to evaluate and accept your PR, we ask that you sign a contribution license agreement. It's all electronic and will take just minutes. I promise there's no faxing. https://cla.azure.com.
    </span>

TTYL, AZPRBOT;

**Comment by azurecla** [comment]
Hi __@bharathsreenivas__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <span>
        In order for us to evaluate and accept your PR, we ask that you sign a contribution license agreement. It's all electronic and will take just minutes. I promise there's no faxing. https://cla.azure.com.
    </span>

TTYL, AZPRBOT;

File: hdinsight-linux-with-hue-application/README.md [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+
+Create HDInsight Linux Cluster with Edge Node and Hue app installed on Edge Node -<br>
+<a href="https://portal.azure.com/#create/Microsoft.Template/uri/https%3A%2F%2Fraw.githubusercontent.com%2FAzure%2Fazure-quickstart-templates%2Fmaster%2Fhdinsight-linux-with-hue-application%2Fazuredeploy.json" target="_blank">
+    <img src="http://azuredeploy.net/deploybutton.png"/>
+</a>
+<a href="http://armviz.io/#/?load=https%3A%2F%2Fraw.githubusercontent.com%2FAzure%2Fazure-quickstart-templates%2Fmaster%2Fhdinsight-linux-with-hue-application%2Fazuredeploy.json" target="_blank">
+    <img src="http://armviz.io/visualizebutton.png"/>
+</a>
+
+Template installs Hue as an Iaas Cluster Application on the edge node of an existing cluster.
```
Comment (singhkay) [inline hdinsight-linux-with-hue-application/README.md]: This readme looks almost the same as that for hdinsight-linux-add-hue-on-existing-cluster template?

File: hdinsight-linux-with-hue-application/azuredeploy.json [hunk 2]
```diff
@@ -0,0 +1,225 @@
...
+        "description": "These credentials can be used to remotely access the cluster."
+      }
+    },
+    "sshPassword": {
+      "type": "securestring",
+      "metadata": {
+        "description": "The password must be at least 10 characters in length and must contain at least one digit, one non-alphanumeric character, and one upper or lower case letter."
+      }
+    },
+    "location": {
```
Comment (singhkay) [inline hdinsight-linux-with-hue-application/azuredeploy.json]: location can be a variable with value resourceGroup().location

Comment (bharathsreenivas) [inline hdinsight-linux-with-hue-application/azuredeploy.json]: we restrict hdinsight edge node clusters to certain regions only.

File: hdinsight-linux-add-hue-on-existing-cluster/azuredeploy.json [hunk 3]
```diff
@@ -0,0 +1,73 @@
+{
+    "$schema": "https://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json#",
```
Comment (singhkay) [inline hdinsight-linux-add-hue-on-existing-cluster/azuredeploy.json]: I suggest you reconsider the name for this tempalte. hdinsight-linux-hue-existing-cluster is better and will fit in the Github column limits

image

File: hdinsight-linux-add-hue-on-existing-cluster/azuredeploy.json [hunk 4]
```diff
@@ -0,0 +1,73 @@
...
+    "contentVersion": "1.0.0.0",
+    "parameters": {
+        "clusterName": {
```
Comment (singhkay) [inline hdinsight-linux-add-hue-on-existing-cluster/azuredeploy.json]: prefix "existing" to parameter name to set the right expectations

**Comment by singhkay** [comment]
lgtm, merged

**Pull request merged**